tools/SetupFlow/DevHome.SetupFlow.ElevatedComponent/Tasks/ElevatedInstallTask.cs

```diff
@@ -117,4 +125,30 @@ private FindPackagesOptions CreateFindOptionsForPackageId(string packageId)
 
         return findOptions;
     }
+
+    /// <summary>
+    /// Find a specific version in the list of available versions for a package.
+    /// </summary>
+    /// <param name="package">Target package</param>
+    /// <param name="version">Version to find</param>
+    /// <returns>Specified version</returns>
+    /// <exception>Exception thrown if the specified version was not found</exception>
+    private PackageVersionId FindVersionOrThrow(ElevatedInstallTaskResult result, CatalogPackage package, string version)

```

Comment:
Ideally we shouldn't need to re-implement this method here. However, because the elevated project code is copied from the non-elevated project, this method had to be copied here as well.

Refactoring the elevated process to eliminate redundant code is tracked here:
 - #622